pkg/system/lstat_windows.go

```diff
@@ -1,6 +1,20 @@
 package system // import "github.com/docker/docker/pkg/system"
 
-import "os"
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+const reparsePointNameSurrogate = 0x20000000
+
+type fileInfo struct {
+	data        os.FileInfo
```

Comment:
You can use embed `os.FileInfo` anonymously, i.e.

```go
type fileInfo struct {
￼ 	os.FileInfo
	maskReparse bool
￼ }
```

so you won't need to add `.data` everywhere else.